test-pup/react-client-scr/data-table/data-table-filters.test.js

```diff
@@ -0,0 +1,809 @@
+const {login} = require("../../common/login-to-scr");
```

Comment:
May be it make sense to split file. As for me - length is too big. Also several IDEA warning could be fixed.